store/sqlstore/emoji_store.go

```diff
@@ -183,19 +183,18 @@ func (es SqlEmojiStore) Delete(id string, time int64) store.StoreChannel {
 	})
 }
 
-func (es SqlEmojiStore) Search(name string, prefixOnly bool, limit int) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		var emojis []*model.Emoji
+func (es SqlEmojiStore) Search(name string, prefixOnly bool, limit int) ([]*model.Emoji, *model.AppError) {
+	var emojis []*model.Emoji
 
-		term := ""
-		if !prefixOnly {
-			term = "%"
-		}
+	term := ""
+	if !prefixOnly {
+		term = "%"
+	}
 
-		term += name + "%"
+	term += name + "%"
 
-		if _, err := es.GetReplica().Select(&emojis,
-			`SELECT
+	if _, err := es.GetReplica().Select(&emojis,
+		`SELECT
 				*
```

Comment:
Can you remove 1 tab indentation here to the rest of the query to maintain it consistent with the previous existing indentation?

Reply:
Done.